### DISCUSSION TITLE
dark mode

### DISCUSSION THREAD

**Discussion by tigawanna** [cschroeter/park-ui#360] [Q&A] [answered]
Hi uys , so far ma loving this lib but ma a bit lost on how to do dark mode with tailwind because adding the dark class resets the accent color to the defaault , is this expected behaviour and is ther a work aourn to ensure the dark version of accent is what's availabe when dark mode toggled llike bg and fg

**Comment by cschroeter** [comment, marked as answer]
With a heavy heart, I have decided to discontinue support for Tailwind as the initial maintainer no longer has time to maintain it.

**Reply by tigawanna** [reply]
💔. but understandable. would really appreciate the code for the tailwind plugin if possible
